### ISSUE TITLE
[Feature Request] Flipping the board without changing the color you play

### ISSUE THREAD

**Issue by wadlo** [TowelSniffer/Anki-Chess-2.0#43]
I'd like to practice playing as black while looking at white's viewpoint, and vice versa. My use case for this is that if I make a mistake, I want to practice "Seeing" what my opponent could do to punish my mistake. Thus playing as my opponent, but from my viewpoint.

There is a "flip" variable, however it doesn't just flip visuals. It also flips which player you're playing as. My feature request is adding a "flipVisuals" property that only flips the visuals and lets you play as the opposite side's pieces. Happy to create the PR myself if you feel this would be a good addition to Anki-Chess.

**Comment by TowelSniffer** [comment]
Yeah this wouldn't be hard at all. Should be able to add the variable on next release

**Comment by TowelSniffer** [comment]
yeah I can include this as a set variable easily enough. When you say an input to flip the board, would this be like a keyboard shortcut, because that should also be easy enough. I don't really want to add something like a button though so I don't know how it would work on mobile. 

Thinking about it a little, the random option probably isn't the way with chess. Go is much more symmetrical so this feature made sense in that template but it just ends up with kings and queens on the wrong squares with chess. I have been busy the last few days but i should hopefully be able to push an update by Tuesday.

**Comment by TowelSniffer** [comment]
@wadlo regarding you initial request of having the option the the board being played from the reverse side. I am hesitant to add this feature for two reasons. First, I want to limit unnecessary features to keep the templates as simple as possible. This just makes it easier to maintain and document. And second, I really don't think it is all that beneficial in the first place. With this in mind I am closing this for now, but feel free to let me know your thoughts.

**Comment by jpfreiire** [comment]
Hi TowelSniffer, I share the same opinion from wadlo, maybe would be good to add such a keyboard shortcut to flip slides, because i would like to train for openings with both black and white peaces, like guess the best moves in both black and white pov. Dear TowelSniffer, thanks for your efforts.
